server/src/fat/java/com/ibm/ws/lars/rest/ApiTest.java

```diff
@@ -1202,6 +1206,81 @@ public void test500Mapping() throws InvalidJsonAssetException, ParseException, I
         mongoClient.close();
     }
 
+    @Test
+    public void testRuntimeException() throws ClientProtocolException, IOException {
```

Comment:
This is correct, I've moved the error servlet and given it a different name so it's easier to see that the tests are hitting different endpoints.